lib/model.php

```diff
@@ -163,18 +163,20 @@ protected function set_table_name( $class_name ) {
 	 *
 	 * @param string      $class_name    The target class name.
 	 * @param string      $property      The property to get the value for.
-	 * @param string|null $default_value Default value when property does not exist.
+	 * @param mixed|null $default_value Default value when property does not exist.
```

Comment:
```suggestion
	 * @param mixed|null  $default_value Default value when property does not exist.
```

Reply:
`mixed|null` is an oxymoron - `null` is already included in `mixed`.